filament/src/MaterialInstance.cpp

```diff
@@ -76,9 +76,15 @@ FMaterialInstance::FMaterialInstance(FEngine& engine, FMaterial const* material)
 void FMaterialInstance::initDefaultInstance(FEngine& engine, FMaterial const* material) {
     mMaterial = material;
 
+    const RasterState& rasterState = mMaterial->getRasterState();
```

Comment:
If I remember correctly, I think we've had issues here before changing this logic in the constructor but not here or vice versa. I wonder if we could refactor this function and the `FMaterialInstance` constructor to call a helper function instead to avoid duplicating this logic.

Reply:
Good idea, I added a commit that renames and repurposes `initParameters()` for this.